media-libs/gegl/gegl-0.2.0-r3.ebuild

```diff
@@ -12,7 +12,7 @@ inherit versionator gnome2-utils eutils autotools #vala
 
 DESCRIPTION="A graph based image processing framework"
 HOMEPAGE="http://www.gegl.org/"
-SRC_URI="ftp://ftp.gimp.org/pub/${PN}/${PV:0:3}/${P}.tar.bz2"
+SRC_URI="https://download.gimp.org/pub/${PN}/${PV:0:3}/${P}.tar.bz2"
 
 LICENSE="|| ( GPL-3 LGPL-3 )"
 SLOT="0"
```

Comment:
this is a stable ebuild, revbump it instead

Reply:
I was under the impression renaming the ebuild does a revbump? In either case someone already pushed a -r4, so not sure if this PR is still needed.

Reply:
> I was under the impression renaming the ebuild does a revbump?

It does, but let's make a copy so that if it turns out worse, people can re-try with the previous version. Moves are for non-stable ebuilds that had no chance to compile, in my eyes.

> In either case someone already pushed a -r4, so not sure if this PR is still needed.

r4 has EAPI 6 but not this patch for ffmpeg. It would be cool if you could turn the merge request to add r5 with that patch based on r4.